services/self_hosted.py

```diff
@@ -83,9 +91,15 @@ def license_seats() -> int:
     return license.number_allowed_users or 0
 
 
+async def enterprise_has_seats_left() -> bool:
+    owners = await activated_owners_async()
+    count = await sync_to_async(owners.count)()
+    return count < license_seats()
+
+
 def can_activate_owner(owner: Owner) -> bool:
     """
-    Returns true iff there are available seats left for activation.
+    Returns true if there are available seats left for activation.
```

Comment:
weird how all of these had 2 "f"s lol
Maybe they meant if and only if?

Reply:
yeah is that a shorthand I don't know about either?

Reply:
[microsoft says](https://github.com/microsoft/terminal/issues/4010) it's if and only if but `iff` only works with 2 statements right? so i think typo